src/main/java/com/pokegoapi/google/common/geometry/S2EdgeIndex.java

```diff
@@ -529,7 +529,7 @@ private void getEdgesInChildrenCells(S2Point a, S2Point b, List<S2CellId> cover,
     }
   }
 
-  /*
+  /**
```

Comment:
I ran everything against check style, spaces look the same but I added an extra * to make it a docblock comment